### DISCUSSION TITLE
Limit Decimals in Surburst Chart while hovering #superset

### DISCUSSION THREAD

**Discussion by Kamakshi12** [apache/superset#18446] [Ideas]
## Screenshot
![image](https://user-images.githubusercontent.com/78851183/107870121-e7184800-6ebb-11eb-855b-2cef232e84e5.png)

## Description
I would like to limit the decimals to 2 places on the surburst chart while hovering on it. Kindly suggest ways to accomplish the same.

**Comment by NikitaVladimirov1985** [comment]
Hello!
Using
ROUND(your_function,2)
